src/server/models/list.ts

```diff
@@ -33,18 +25,16 @@ export async function findUserLists(
       OR "jsonData"->'administrators' @> '"${emailAddress}"'
       ORDER BY id ASC
     `;
-
-    const lists = await prisma.$queryRaw(query);
+    // @ts-ignore
+    const lists: List[] = await prisma.$queryRawUnsafe(query);
```

Comment:
Hey Ali, this is a good question. I didn't do a good job of explaining this in the PR. 

## $queryRaw``

So I've upgraded Prisma to version 3.0.1 and there are some breaking changes with **$queryRaw**.

You can read more about it in [the documentation here:](https://github.com/prisma/prisma/releases?page=4)

<img width="853" alt="Screenshot 2022-09-27 at 13 29 45" src="https://user-images.githubusercontent.com/1377253/192526875-6e597d87-b3a3-49d2-9c98-6a4c035d28a3.png">

---

The way we were using queries before in version 2x of Prisma **prisma.$queryRaw(query);** was apparently UNSAFE (see image above)

So I've updated **$queryRaw()** to **$queryRaw``** where I can, however there have been some scenarios that it hasn't worked. To fix this I replaced **$queryRaw()** with **$queryRawUnsafe()** which is something that was introduced in Prisma 3x.

## $queryRawUnsafe()

To my understanding if a query has variables being passed into it in a certain way:
```js
const name = 'Bob'
await prisma.$queryRaw`SELECT 'My name is ${name}';`
```

This will not work with **$queryRaw``** but will with **$queryRawUnsafe()**. My understanding is that  **$queryRawUnsafe()**. in 3x and **$queryRaw()** in 2x are identical but the 'unsafeness' is made more explicit.


## Going forward
I guess it would make a sense to create a ticket to re-write our unsafe queries in a way that makes more sense. 

Or that could be done in this ticket?

Reply:
@alisalaman-cyb/ @RichardBray. It is about the same effort for either.

- If you want to keep the `prisma.$queryRaw()` call, instead of unsafe, you have to do 
```
await prisma.$queryRaw(
  Prisma.sql`...`
)
```
- Or use 
```
await prisma.$queryRaw`SELECT 'My name is ${name}';`
```

Reply:
Hmmm, I feel like there whole query will need to be re-written to make it more safe, I don't know if it's as simple as:
```
await prisma.$queryRaw(
  Prisma.sql`...`
)
```

But I can look into it

Reply:
indeed. so these are one of the considerations you need to make when refactoring code. I suppose this goes more into engineering (rather than plain old developing).

So, do you want to explicitly label things as `queryRawUnsafe`? What's the benefit of that? (You've already answered)

but then like @alisalaman-cyb has commented, it's changed our Prisma calls. would that effect the tests that may be spying on these calls? But you might consider if it causes too much churn in the code (code keeps changing after it's merged in). 

Then you have to consider how likely are they going to deprecate the unsafe call or the wrapped `Prisma.sql` statement.. 

imho, doesn't matter either way. this sort of change is fairly inconsequential. whatever is easiest 🙂